File: imt.cpp
#include <iostream>
using namespace std;
int main()
{
	float height;
	float weight;
	cout << "Enter your height:";
	cin >> height;
	cout << "Enter your weigth:";
	cin >> weight;
	float m = height/100;
	float result = weight/(m*m);
	cout << "----------\n";
	if (result < 18.5)
	{
		cout << "You are underweight";
	}
	if (result > 18.5 && result < 24.9)
	{
		cout << "You have a normal weight";
	}
	if (result > 29.9 && result < 29.9)
	{
		cout << "You have a overweight";
	}
	if (result > 29.9)
	{
		cout << "You are obese";
	}
}

File: imt.java
import java.util.Scanner;
public class imt {
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        System.out.println("Введите свой рост:");
        double height = scan.nextInt();
        System.out.println("Введите свой вес:");
        double weight = scan.nextInt();
        double m = height/100;
	    double result = weight/(m*m);
        if (result < 18.5)
        {
            System.out.println("У вас недостаточный вес");
        }
        if (result > 18.5 && result < 24.9)
        {
            System.out.println("У вас нормальный вес");
        }
        if (result > 24.9 && result < 29.9)
        {
            System.out.println("У вас избыточный вес");
        }
        if (result > 29.9)
        {
            System.out.println("У вас ожирение");
        }
    }
}
